src/Assignment5/MathApp/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace MathApp
+{
+    /// <summary>
+    /// Progran class has the main method.
+    /// </summary>
+    internal class Program
```

Comment:
This program class is not required as this should be a output library.

Reply:
The project is changed to output library and the program class is removed.